src/aspire/operators/polar_ft.py

```diff
@@ -2,50 +2,60 @@
 
 import numpy as np
 
-from aspire.basis import Basis
-from aspire.nufft import anufft, nufft
+from aspire.image import Image
+from aspire.nufft import nufft
 from aspire.utils import complex_type
 
 logger = logging.getLogger(__name__)
 
 
-class PolarBasis2D(Basis):
+class PolarFT:
     """
     Define a derived class for polar Fourier representation for 2D images
     """
 
     def __init__(self, size, nrad=None, ntheta=None, dtype=np.float32):
         """
-        Initialize an object for the 2D polar Fourier grid class
+        Initialize an object for the 2D polar Fourier grid class. `PolarFT` expects that
+        images are real and uses only half of the `ntheta` values. Downstream algorithms should
+        take advantage of the conjugate symmetry of the polar Fourier coefficients if the full
+        set is needed.
 
         :param size: The shape of the vectors for which to define the grid.
```

Comment:
We haven't defined what the “grid” is here (also below, under `dtype`). It should either be clarified or replaced with some other word (here “transform” would work).

Reply:
Sounds good. Done.